feat: add gradle.properties and update jenkinsfile to push image when the master branch change

kimbyungnam opened the pull request.

kimbyungnam committed:
JIB initial setup

blupine commented on core/src/main/java/com/bbolab/gaonna/core/service/FileService.java:
```diff
@@ -0,0 +1,41 @@
+public class FileService {
+    private final FileRepository fileRepository;
+
+    public File create(byte[] bytes) {
```
반환형이 File 타입인데 파일 생성후에도 null이 반환되는거 아닌가요?

kimbyungnam committed:
feat: add gradle.properties files and update jenkinsfile to push docker image when the master branch change

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

kimbyungnam committed:
fix: Jenkinsfile syntax error

LeeMinki commented:
LGTM
